packages/api-generator/src/types.ts

```diff
@@ -290,6 +301,13 @@ function generateDefinition (node: Node<ts.Node>, recursed: string[], project: P
   const tc = project.getTypeChecker()
   type = type ?? node.getType()
 
+  if (type.getAliasSymbol()?.getName() === 'NonNullable') {
```

Comment:
`getAliasSymbol` is called at least 3 times in this file. Could probably be simplified.

Reply:
Parsing is inherently complicated.

The second one could probably change to `symbol?.getName()`, although idk how that behaves if `type.getSymbol()` is hit.